packages/migrate/src/__tests__/__snapshots__/MigrateDiff.test.ts.snap

```diff
@@ -1,84 +1,50 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`migrate diff generic should fail when --from-local-d1 is used with --shadow-database-url 1`] = `
```

Comment:
What happened to this snapshot?

Reply:
It's not interesting to capture the full output here, it makes it hard to change the help output because so many snapshots rely on it
So I changed this to check that it throws.

Reply:
See new commit [`07a9aa7` (#23715)](https://github.com/prisma/prisma/pull/23715/commits/07a9aa769b13acc04b34f392dff6e1e7a5fb414d)